entity-service-attribute-translator/src/main/java/org/hypertrace/entity/attribute/translator/EntityAttributeMapping.java

```diff
@@ -22,6 +28,9 @@ public class EntityAttributeMapping {
   private static final String SCOPE_PATH = "scope";
   private static final String ATTRIBUTE_PATH = "attribute";
   private static final String NAME_PATH = "name";
+  private static final Set<AttributeKind> MULTI_VALUED_ATTRIBUTE_KINDS =
+      Set.of(
+          TYPE_STRING_ARRAY, TYPE_INT64_ARRAY, TYPE_DOUBLE_ARRAY, TYPE_BOOL_ARRAY, TYPE_STRING_MAP);
```

Comment:
Maps aren't really multivalued in the same ways as arrays. They support different operators, and for example later you write

```java
    if (entityAttributeMapping.isMultiValued(attributeKind)) {
      suffixedSubDocPath = DOT_JOINER.join(subDocPath, VALUE_LIST_KEY, VALUES_KEY);
```

That would fail for a map, no?

Reply:
Good catch, Aaron. I'll remove the map from here.